src/services/zapper.ts

```
@@ -31,9 +31,14 @@ export class ZapperService extends Service {
       .then(handleHttpError)
       .then(res => res.json());
     const network = Chains[this.chainId] ?? "ethereum";
```

With this new card in mind, do we want to default the network to Ethereum?

---

Think we can remove that default, and set chainId on constructor. Since zapper is still just supported in mainnet, we can throw an error if its initialized on another network.

---

Removed the default with https://github.com/yearn/yearn-sdk/pull/245/commits/96db71cd49e2af1607c89f3518ed73efd4d20c66 and then refactored so we throw when the other zapper is instantiated with any other chain id with https://github.com/yearn/yearn-sdk/pull/245/commits/b974dd20a93ebaa6887af1a06c4ed3a49172ba0d

---

Unfortunately this is not resolved, I think the `token.spec.ts` was expecting the fantom network to be supported, but it is not, right?

Is this fix https://github.com/yearn/yearn-sdk/pull/245/commits/c32ccc042f24b934a96bd22eca338ca8833a05f1 what we should be having?